odfuzz/odfuzz.py

```diff
@@ -20,6 +21,7 @@
 
 
 def main():
+    subprocess.check_call("mongod > /dev/null 2>&1 &", shell=True)
```

Comment:
I am not sure this is a good idea. What about to employ supervisord and provide a proper way how to start processes?

Reply:
I dont't think this should be in the main odfuzz.py, especially when this would not work on Windows environment.

The change for dockerfile is triggered by call for better usage in internal SAP tooling, but odfuzz should work correctly as standalone tool as well.

Reply:
I see it doesn't work as a solution. However I don't think the current state is good. (Starting the db as part of an entry point shell command, which will get overwritten by providing arguments to `docker run`). Another way could be to include an entry point script which launches the db and then either odfuzz or a shell, depending on what you want.

But since the current state is working for the tooling for now I will leave this up to you to decide.